### DISCUSSION TITLE
Not able to upload file of size 7 mb from the piccolo admin.

### DISCUSSION THREAD

**Discussion by sumitsharansatsangi** [piccolo-orm/piccolo_admin#285] [General]
I am able to try to upload video of some kb  (97 kb) and it is uploading , but when I am trying to upload video of some mb (7 mb) , then it is giving the error `file size is too large` , I am not able to find the possible reason. 
Is there any constraints in the piccolo admin?

**Comment by dantownsend** [comment]
If running Piccolo Admin behind a reverse proxy like Nginx, make sure that Nginx is configured to accept larger file sizes.

I don't think Piccolo Admin and uvicorn impose any limits on the file size.

**Reply by sumitsharansatsangi** [reply]
Ok, Let me see on the nginx.

**Reply by sumitsharansatsangi** [reply]
Is piccolo admin not using presigned_url concept?

**Reply by dantownsend** [reply]
No, Piccolo Admin signs the URLs when the file is requested.

**Reply by sumitsharansatsangi** [reply]
You were true.  `By default, NGINX® has a upload limit of 1 MB per file.`
